SpywareFilter/sections/specific.txt

```diff
@@ -35,6 +35,7 @@
 ||home.ebs.co.kr/common/js/wlo.js
 ||newzit.com/setABframe.html
 ||seoul.co.kr/weblog/
+||hqq.tv/js/video.counters.js
```

Comment:
We have old rule `||hqq.watch/js/video.counters.js`. It seems `hqq.watch` is dead now, so, remove it.